CHANGELOG.md

```diff
@@ -87,6 +87,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 #### Fixed
 
 - The auditor now uses width-first tracking, to bring it in alignment with the new wallet.
+- Optimize auditor tracking by not to re-attempt fetched spend.
```

Comment:
Hi Qi,

Sorry, but you need to create a new entry. It shouldn't be appended to one of the older ones. It should have today's date on it.